.circleci/config.yml

```diff
@@ -27,6 +27,7 @@ jobs:
           STRATEGY_LOCAL: "true"
           CANONICAL_URL: "http://localhost:3000"
           WAIT_HOSTS: localhost:5432, localhost:6379
+          DISABLE_FILE_UPLOADS: "true"
```

Comment:
This was the thing blocking the server from starting...
Essentially borking the test workflow.
Its my bad sry, i've made some changes on the server side...

Reply:
lol so simple 😅 - thanks for the fix!

Reply:
do you know why the dev container might be behaving differently (i think it's getting and older server version?) when the config looks just like the circle ci one?